voting/orbs/Elections/exports_unsafetests.go

```diff
@@ -11,26 +11,28 @@ package elections_systemcontract
 import (
 	"fmt"
 	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1"
+	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/env"
 	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/ethereum"
 	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath/safeuint64"
 	"time"
 )
 
-var PUBLIC = sdk.Export(getTokenEthereumContractAddress, getGuardiansEthereumContractAddress, getVotingEthereumContractAddress, getValidatorsEthereumContractAddress, getValidatorsRegistryEthereumContractAddress,
-	unsafetests_setTokenEthereumContractAddress, unsafetests_setGuardiansEthereumContractAddress,
+var PUBLIC = sdk.Export(getTokenEthereumContractAddress, getStakingEthereumContractAddress, getGuardiansEthereumContractAddress, getVotingEthereumContractAddress, getValidatorsEthereumContractAddress, getValidatorsRegistryEthereumContractAddress,
```

Comment:
the whole file is called unsafe

Reply:
I mean to say that if you had a single version with the build tags inline it would be simpler to read - as it is i have to diff it with the safe version to sniff out the differences..